### PR TITLE
Use env variable instead for minio release images.

### PR DESCRIPTION
**Pull request by anurags92**
This is heavily based on changes in: https://github.com/dgraph-io/dgraph/pull/8503

### CODE REVIEW

File: t/Makefile [hunk 1]
```diff
@@ -26,8 +26,12 @@ test:
 # 	build the t.go binary
 	@go build .
 #	clean go testcache
-	go clean -testcache
-#	run the tests
-	@./t --skip systest/backup,systest/online-restore,systest/loader,tlstest
+	@go clean -testcache
+#	run t.go with specified arguments; otherwise run standard suite
+	@if [ -n "$(args)" ]; then \
```
Comment (mangalaman93) [inline t/Makefile:31]: do we need this `if..else`

File: Makefile [hunk 2]
```diff
@@ -61,8 +61,8 @@ uninstall:
 	@echo "Uninstalling Dgraph ..."; \
 		$(MAKE) -C dgraph uninstall; \
 
-test: image-local
-	@mv dgraph/dgraph ${GOPATH}/bin
+test: docker-image
```
Comment (mangalaman93) [inline Makefile:64]: check if this is required

File: systest/backup/minio-large/docker-compose.yml [hunk 3]
```diff
@@ -69,7 +67,7 @@ services:
       --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
       --tls "ca-cert=/dgraph-tls/ca.crt; server-cert=/dgraph-tls/node.crt; server-key=/dgraph-tls/node.key; internal-port=true; client-cert=/dgraph-tls/client.alpha3.crt; client-key=/dgraph-tls/client.alpha3.key;"
   minio:
-    image: minio/minio:RELEASE.2020-11-13T20-10-18Z
+    image: minio/minio:${MINIO_IMAGE_ARCH}
```
Comment (mangalaman93) [inline systest/backup/minio-large/docker-compose.yml:70]: can we just append `-arm64` when the arch is arm64?

**Comment by joshua-goldstein** [comment]
Do we want to use this PR or the original https://github.com/dgraph-io/dgraph/pull/8503?  I've incorporated all these changes into my PR as well.

**Comment by anurags92** [comment]
> Do we want to use this PR or the original #8503? I've incorporated all these changes into my PR as well.

@joshua-goldstein Let's use your PR. I'll close this one. Can you take a look at @mangalaman93 's suggestions as well.

**Comment by anurags92** [comment]
Closing it in favour of https://github.com/dgraph-io/dgraph/pull/8503

**Pull request closed without merging**